backend/app/assets/javascripts/spree/backend/views/tables/editable_table_row.js.coffee

```diff
@@ -43,3 +46,16 @@ Spree.Views.Tables.EditableTableRow = Backbone.View.extend
     switch key
       when @ENTER_KEY then @onSave(e)
       when @ESC_KEY then @onCancel(e)
+
+  storeValues: ->
+    @originalFormData = $(":input", @$el).serialize()
+
+    $(":input", @$el).each ->
+      $input = $(this)
+      $input.data 'original-value', $input.val()
+
+  restoreOriginalValues: ->
+    $(":input", @$el).each ->
+      $input = $(this)
+      originalValue = $input.data('original-value')
+      $input.val(originalValue).change()
```

Comment:
Instead of rolling our own set/reset mechanism we should consider to use a [Backbone model](http://backbonejs.org/#Model) like [in the stock management views](https://github.com/solidusio/solidus/blob/master/backend/app/assets/javascripts/spree/backend/stock_management/index_update_forms.coffee)